### PR TITLE
feat: Adds ElementBlock component

### PR DESCRIPTION
**Pull request by chris-decker-volusion**
- Running into a typescript issue on the `defaultConfig` import in `Block.tsx`
- Also getting an `Import of non-existent exports` warning for my types in defined in the `types` file.
- Using placeholders for the `globalSettings` keys as they don't exist yet.
- Haven't bothered with tests until the `globalSettings` keys exist.
- Need to update readme.

### CODE REVIEW

File: src/ElementBlock/.element-block
Comment (cameron-jones-volusion) [inline src/ElementBlock/.element-block]: The `.element-block` file is not used by the component system and can be deleted.

File: src/ElementBlock/src/configs.ts
Comment (cameron-jones-volusion) [inline src/ElementBlock/src/configs.ts]: For backwards compatibility, the default values should have no visible effect on the wrapped blocks. The result of rendering with the default values should be a wrapping `div` element with no styling or attributes.

File: src/ElementBlock/src/getStyles.ts
Comment (cameron-jones-volusion) [inline src/ElementBlock/src/getStyles.ts]: For now, we can deal with the missing global style variables by setting arbitrary fixed values here.
```suggestion
    if (/^small$/i.test(paddingSize)) return '0.5rem';
    if (/^medium$/i.test(paddingSize)) return '1rem';
    if (/^large$/i.test(paddingSize)) return '1.5rem';    
```

File: src/ElementBlock/src/Block.tsx
Comment (cameron-jones-volusion) [inline src/ElementBlock/src/Block.tsx:24]: Using `map` & `reduce` here will get rid of the eslint security warning triggered by this line, something like this:
```tsx
    const attrs = customAttrs.map((attr: { name: string; value: string }) => {
        const hyphenateString = (string = '') => {
            const nonAlphaNumericChars = /[^a-zA-Z0-9]+/g;
            const attr = string
                .trim()
                // .replace(specialChars, '')
                .replace(nonAlphaNumericChars, '-');
            return attr;
        };
        return {
            [`data-element-${hyphenateString(attr.name)}`]: value
        }
    })
    .reduce((acc, attr) => {
        return { ...acc, ...attr }
    }, {})
```

**Review by cameron-jones-volusion — CHANGES_REQUESTED** [review]
In order to get this component out, and unblock related work in the Element Platform, we can use static values as placeholders for the variables that we will eventually be getting from the global styles object. That should be enough to write some tests and get this merged.

File: src/ElementBlock/src/Block.tsx
Comment (chris-decker-volusion) [reply src/ElementBlock/src/Block.tsx:24]: This issue seems to have disappeared...

File: rollup.config.js
Comment (cameron-jones-volusion) [inline rollup.config.js]: Useful for local development, but we should remove before merging.

File: src/index.js
Comment (cameron-jones-volusion) [inline src/index.js]: This component will not be available through `ElementPropTypes.component()`, so it should not be included in the `StandardComponents` object. For now, we should export it from the top level of the module.

**Review by cameron-jones-volusion — CHANGES_REQUESTED** [review]
* Only major change is that we should make ElementBlock a top-level export of the package.
* Remove README for now.

File: rollup.config.js
Comment (chris-decker-volusion) [reply rollup.config.js]: 4ea6b16dfe50d16cf3e9e4a55a38b9eff91fa30e

File: src/index.js
Comment (chris-decker-volusion) [reply src/index.js]: f98d70e34dab56e1b6221b39cb5bf26014fd78fd

**Review by cameron-jones-volusion — APPROVED** [review]

**Pull request merged**
